plugins/nginx-vhosts/functions

```diff
@@ -6,14 +6,29 @@ source "$PLUGIN_AVAILABLE_PATH/config/functions"
 source "$PLUGIN_AVAILABLE_PATH/domains/functions"
 source "$PLUGIN_AVAILABLE_PATH/ps/functions"
 
+get_nginx_location() {
+  declare desc="check that nginx is at the expected location and return it"
+  local NGINX_LOCATION="/usr/sbin/nginx"
+  if ! [ -x "$NGINX_LOCATION" ]; then
```

Comment:
This should be `if [[ -z "$NGINX_LOCATION" ]]; then`

Reply:
How could `$NGINX_LOCATION` empty at that point? It's set right above that. 

Also, what I'm doing here is checking that `niginx` is executable for the current user, not finding out if the variable is empty.